src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -369,7 +369,8 @@ def assign_general_population_guardians(self, pop: pd.DataFrame) -> pd.DataFrame
         # Non-GQ population
         gen_population = pop.loc[~pop["household_id"].isin(data_values.GQ_HOUSING_TYPE_MAP)]
         under_18_idx = pop.loc[
-            (pop["age"] < 18) & (~pop["household_id"].isin(data_values.GQ_HOUSING_TYPE_MAP))
+            (pop["age"] < 18) &
+            (~pop["household_id"].isin(data_values.GQ_HOUSING_TYPE_MAP))
```

Comment:
Formatting

Reply:
It's a matter of taste, but on our repos we try to have the operator start the new line. So:
```
(pop["age"] < 18) 
& (~pop["household_id"].isin(data_values.GQ_HOUSING_TYPE_MAP))

```